actors/builtin/miner/miner_state.go

```diff
@@ -94,8 +94,16 @@ type MinerInfo struct {
 	// Libp2p identity that should be used when connecting to this miner.
 	PeerId peer.ID
 
-	// Amount of space in each sector committed to the network by this miner.
+	// The proof type used by this miner for sealing sectors.
+	SealProofType abi.RegisteredProof
+
+	// Amount of space in each sector committed by this miner.
+	// This is computed from the proof type and represented here redundantly.
 	SectorSize abi.SectorSize
+
+	// The number of sectors in each Window PoSt partition (proof).
+	// This is computed from the proof type and represented here redundantly.
+	WindowPoStPartitionSectors uint64
```

Comment:
I guess its fine to put this here, since we never rewrite this.

Reply:
You could convince me to take it out pretty easily, but if it [the code that derives it] ever changed we'd be in trouble, and this helps make the state transparent.